assets/config/postfix/main.cf

```diff
@@ -88,14 +88,16 @@ virtual_mailbox_maps = {{DB_DRIVER}}:/etc/postfix/database_virtual_mailbox_maps.
 virtual_alias_maps = {{DB_DRIVER}}:/etc/postfix/database_virtual_alias_maps.cf,{{DB_DRIVER}}:/etc/postfix/database_email2email.cf
 
 # set dovecot as delivery service for virtual domains
-virtual_transport = dovecot
+virtual_transport = lmtp:unix:private/dovecot-lmtp
 dovecot_destination_recipient_limit = 1
 
 # =====================================
-# SpamAssassin
+# RSpamD
 # =====================================
-# set SpamAssassin socket
-smtpd_milters = unix:/spamass/spamass.sock
-
+# set RSPAMD socket
+smtpd_milters = inet:127.0.0.1:11332
```

Comment:
Use unix socket for this?
I think rspamd should support it: https://rspamd.com/doc/workers/